Document/0x05a-Platform-Overview.md

```diff
@@ -355,7 +355,7 @@ To improve security and privacy, a Local Broadcast Manager is used to send and r
 
 Broadcast Receivers are components that allow apps to receive notifications from other apps and from the system itself. With it, apps can react to events (internal, initiated by other apps, or initiated by the operating system). They are generally used to update user interfaces, start services, update content, and create user notifications.
 
-Broadcast Receivers must be declared in the Android Manifest file. The manifest must specify an association between the Broadcast Receiver and an intent filter to indicate the actions the receiver is meant to listen for. If Broadcast Receivers aren't declared, the app won't listen to broadcasted messages. However, apps don’t need to be running to receive intents; the system starts apps automatically when a relevant intent is raised.
+There are two ways to make a Broadcast Receiver known to the system. One way is to declare it in the Android Manifest file. The manifest should specify an association between the Broadcast Receiver and an intent filter to indicate the actions the receiver is meant to listen for.

```

Comment:
Can you explain the difference between explicit and implicit broadcast receivers here please?

Reply:
also consider referencing https://developer.android.com/guide/components/broadcasts.html#security-and-best-practices

Reply:
The only reference I could find (and added) about "implicit broadcast" in Android Dev is under [Manifest-declared receivers](https://developer.android.com/guide/components/broadcasts.html#manifest-declared-receivers):

> implicit broadcasts (broadcasts that do not target your app specifically)

My understanding for "implicit broadcast receiver" is that it doesn't only listen for broadcasts that target the app itself, but also for those "implicit broadcasts". However, there's no reference I could find in the documentation.

Reply:
@cpholguera Is there a particular point you think should be imported from the "Security considerations and best practices" paragraph, or are you suggesting adding the whole page as a reference?

Reply:
@zehuanli these are implicit broadcasts: https://developer.android.com/guide/components/broadcast-exceptions, basically most of them are system events where an app can listen to and act upon.

Reply:
At least referencing to that page as a link should be sufficient, thank you :)

Reply:
Thanks!

Reply:
Link to  [Security considerations and best practices](https://developer.android.com/guide/components/broadcasts.html#security-and-best-practices) has been added.

@commjoen description of "implicit broadcast" has been added in [commit](https://github.com/zehuanli/owasp-mstg/commit/72694f066d156e02cce65f1fef837a9fad605564). Changes of implicit broadcasts in [Broadcast Exceptions](https://developer.android.com/guide/components/broadcast-exceptions) has been added in [commit](https://github.com/zehuanli/owasp-mstg/commit/441e0a699dbd61d0696fb9d618af99f23f0e25ca). Please let me know if there are any changes needed.